charts/azimutt/values.yaml

```diff
@@ -0,0 +1,173 @@
+azimutt:
```

Comment:
If I want to add/remove/renames values here, will it breaks everyone deployment?